azurerm/internal/services/search/resource_arm_search_service.go

```diff
@@ -108,6 +109,15 @@ func resourceArmSearchService() *schema.Resource {
 				Default:  true,
 			},
 
+			"ip_rules": {
```

Comment:
would this be better as 
```suggestion
			"allowed_ips": {
```

Reply:
Done!